skimage/segmentation/_felzenszwalb_cy.pyx

```diff
@@ -125,3 +125,118 @@ def _felzenszwalb_grey(image, double scale=1, sigma=0.8,
         flat = flat[flat]
     flat = np.unique(flat, return_inverse=True)[1]
     return flat.reshape((height, width))
+
+
+def _felzenszwalb_rgb(image, double scale=1, sigma=0.8,
```

Comment:
@sciunto : Sigma is only used to pass into `ndimage.gaussian_filter`, consequently there should be no real gain to typing it.